java-wrapper/src/main/java/org/ballerinalang/twitter/NativeHttpToTwitterAdaptor.java

```diff
@@ -0,0 +1,102 @@
+package org.ballerinalang.twitter;
```

Comment:
Folks, don't we need to license header here ?

Reply:
Noted. Lets add license header here

Reply:
Resolved the missing license header issue with https://github.com/ballerina-platform/module-ballerinax-twitter/pull/28